go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/pinecone-io/go-pinecone
 go 1.21
 
 require (
-	github.com/golang/protobuf v1.5.4
```

Comment:
I ran `go mod tidy` locally and this was removed... I think the `google.golang.org/protobuf v1.34.1` below covers it?

Reply:
Yeah, we should prefer google.golang.org according to https://protobuf.dev/reference/go/faq/.